packages/core-snapshots-cli/lib/db/queries/index.js

```diff
@@ -0,0 +1,29 @@
+const loadQueryFile = require('../utils/load-query-file')
+
+module.exports = {
+  blocks: {
+    heightRange: loadQueryFile(__dirname, './blocks/height-range.sql'),
+    latest: loadQueryFile(__dirname, './blocks/latest.sql'),
+    findByHeight: loadQueryFile(__dirname, './blocks/find-by-height.sql'),
+    deleteFromHeight: loadQueryFile(__dirname, './blocks/delete-from-height.sql')
+  },
+  transactions: {
+    timestampRange: loadQueryFile(__dirname, './transactions/timestamp-range.sql'),
+    deleteFromTimestamp: loadQueryFile(__dirname, './transactions/delete-from-timestamp.sql')
+  },
+  rounds: {
+    deleteFromRound: loadQueryFile(__dirname, './rounds/delete-from-round.sql')
+  },
+  transactionsExport: (start, end) => {
```

Comment:
All SQL queries should be in files and be loaded via `loadQueryFile` to make sure we get all benefits of https://vitaly-t.github.io/pg-promise/QueryFile.html.

Reply:
I agree, wanted to move all queries there, but they don't work with streams, at least I couldn't find a way of getting parsed query to raw text from loadQuery format, as it is parsed on the pg-server. The querystream uses custom pg-native implementation and that is why only string is accepted there as it forwards it to the cursor.